pkg/secretless/config/v2/v1_service.go

```diff
@@ -2,6 +2,8 @@ package v2
 
 import (
 	"fmt"
+	validation "github.com/go-ozzo/ozzo-validation"
+	"gopkg.in/yaml.v2"
 	"sort"
 	"strings"
```

Comment:
Import list sorting needed

Reply:
@doodlesbykumbi You can set your IDE to automatically do this for you.